pkg/rancher/migrations.go

```diff
@@ -86,3 +87,22 @@ func forceUpgradeLogout(configMapController controllerv1.ConfigMapController, to
 
 	return err
 }
+
+// forceSystemAndDefaultProjectCreation will set the correcsponding conditions on the local cluster object,
+// if it exists, to Unknown. This will force the corresponding controller to check that the projects exist
+// and create them, if necessary.
+func forceSystemAndDefaultProjectCreation(clusterClient v3.ClusterClient) error {
+	localCluster, err := clusterClient.Get("local", metav1.GetOptions{})
```

Comment:
We are going to do this on every startup and leader change which I don't think is the intention. forceUpgradeLogout sets a value on a configmap so it's a 1 time operation which I think this should also be.

Reply:
Done.